go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.5
 
 require (
 	github.com/deckarep/golang-set/v2 v2.6.0
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240712162033-89bd3351ce6e
+	github.com/smartcontractkit/chainlink-common v0.2.1-0.20240717132349-ee5af9b79834
```

Comment:
Why is this bump being done?

Reply:
To match the version used in `ccip`. Not strictly necessary for this PR.

Reply:
Oh interesting

Reply:
Did not know we're behind. If it matches ccip then we should be ok but things might go haywire if we match ccip but not chainlink 😛